z-sgov/z-sgov-slovník.ttl

```diff
@@ -1,19 +1,26 @@
+@prefix :     <https://slovník.gov.cz/základní/slovník#> .
 @prefix owl:  <http://www.w3.org/2002/07/owl#> .
+@prefix rdf:  <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml:  <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd:  <http://www.w3.org/2001/XMLSchema#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@base         <https://slovník.gov.cz/základní/slovník> .
 
 <https://slovník.gov.cz/základní/slovník>
-    a                                                     owl:Ontology ;
-    owl:imports                                           <https://slovník.gov.cz/základní/glosář/verze/1.0.1>,
-                                                          <https://slovník.gov.cz/základní/model/verze/1.0.0> ;
+    rdf:type                                              owl:Ontology ;
     owl:versionIRI                                        <https://slovník.gov.cz/základní/slovník/verze/1.0.1> ;
-    rdfs:label                                            "Základní slovník (Z-SGoV)"@cs,
-                                                          "Basic Vocabulary (Z-SGoV)"@en ;
-    <http://purl.org/dc/elements/1.1/description>         "Základní slovník popisuje základní vztahy."@cs ;
-    <http://purl.org/dc/elements/1.1/rights>              <https://creativecommons.org/licenses/by-sa/4.0> ;
+    owl:imports                                           <https://slovník.gov.cz/základní/model/verze/1.0.1>,
+                                                          <https://slovník.gov.cz/základní/glosář/verze/1.0.1> ;
     <http://purl.org/dc/elements/1.1/title>               "Základní slovník (Z-SGoV)"@cs,
                                                           "Basic Vocabulary (Z-SGoV)"@en ;
-    <http://purl.org/dc/terms/created>                    "03.09.2018" ;
-    <http://purl.org/ontology/bibo/status>                "Specifikace"@cs, "Specification"@en ;
+    <http://purl.org/vocab/vann/preferredNamespaceUri>    "https://slovník.gov.cz/základní/pojem/" ;
+    rdfs:label                                            "Základní slovník (Z-SGoV)"@cs ;
+    <http://purl.org/ontology/bibo/status>                "Specification"@en ;
+    rdfs:label                                            "Basic Vocabulary (Z-SGoV)"@en ;
```

Comment:
nerozumím tomu, proč to rozdělovat do dvou statementů

Reply:
ani ja - asi nejaky artefakt toho refaktoringu - delal jsem tam toho vic ... ja to spojim dohromady